### PR TITLE
Move comments, systemtags and updatenotifications to PSR-4

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @rullzer

### CODE REVIEW

File: lib/public/IServerContainer.php [hunk 1]
```diff
@@ -519,4 +519,16 @@ public function getShareManager();
...
+	 * @return \OCP\IDateTimeZone
+	 * @since 9.1.0
+	 */
+	public function getDateTimeZone();
+
+	/**
+	 * @return \OCP\IDateTimeFormatter
+	 * @since 9.1.0
+	 */
+	public function getDateTimeFormatter();
```
Comment (MorrisJobke) [inline lib/public/IServerContainer.php]: Why in here?

Comment (nickvergessen) [inline lib/public/IServerContainer.php]: hmpf, unintentional

Comment (nickvergessen) [inline lib/public/IServerContainer.php]: reverted

**Comment by rullzer** [comment]
:+1:

**Comment by MorrisJobke** [comment]
Looks good and still works 👍

**Pull request merged**